src/js/gamestate.js

```diff
@@ -1,9 +1,11 @@
 /* 
-   The state of a ccg type card game.
+   The state of a ccg type card game, specifically Spacetime.
```

Comment:
I mean, is this something that is generalizable?

Should we have a gamestate or subclasses that might support things more like magic or hearthstone or pokemon or jihad, realtime and not, with expandable cards?

The comment be made to say "The state of a Spacetime game."